solutions/usaco-1015.mdx

```diff
@@ -226,4 +226,56 @@ public class Triangles {
 
 </JavaSection>
 
+<PySection>
+
+```py
+MAXIMUM = 10000
```

Comment:
i'd rename this to just MAX

Reply:
I would rather not. can i please not i like maximum.

Reply:
how, it's literally the same thing

Reply:
pls let me keep it im begging you

Reply:
what do you have against changing it??

Reply:
everything let me keep it.